dietpi/dietpi-software

```diff
@@ -5496,6 +5504,23 @@ Package: wireguard wireguard-dkms wireguard-tools\nPin: release n=bullseye\nPin-
 
 		fi
 
+		software_id=189 # VSCodium
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+			
+			# Install APT repo key
+			G_EXEC eval "curl -sSfL 'https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/raw/master/pub.gpg' | gpg --dearmor -o /etc/apt/trusted.gpg.d/dietpi-vscodium.gpg --yes"
+
+			# Install APT repo list
+			G_EXEC eval "echo 'deb https://paulcarroty.gitlab.io/vscodium-deb-rpm-repo/debs vscodium main' > /etc/apt/sources.list.d/vscodium.list"
```

Comment:
Let's use `dietpi-vscodium.list`. I know we skip the prefix for most lists somehow, but I aim to use it consequently to make identifying easily which files are coming from us and which were created manually or by other installers/packages.